### ISSUE TITLE
o-teaser contrast issues on focus

### ISSUE THREAD

**Issue by notlee** [Financial-Times/origami#918]
Update the `.o-teaser__tag` focus style to match `.o-teaser__heading` within `.o-teaser--hero-image`:

![Screenshot 2022-12-08 at 17 20 59](https://user-images.githubusercontent.com/10405691/206696039-2325243c-dfe0-4552-ba00-355167ceec48.png)

In addition correct the `.o-teaser__tag-suffix` focus style:

![Cursor_and_Climate_Capital___Financial_Times](https://user-images.githubusercontent.com/10405691/206696077-8e9b5741-3151-43b5-ac87-5b24590277e8.png)

https://financialtimes.slack.com/archives/C02FU5ARJ/p1670520671818959?thread_ts=1670511833.076289&cid=C02FU5ARJ

**Comment by notlee** [comment]
In the above examples the copy should be white with a white focus ring
